scipy/interpolate/_fitpack_py.py

```diff
@@ -564,6 +567,97 @@ def spalde(x, tck):
     .. [3] Dierckx P. : Curve and surface fitting with splines, Monographs on
        Numerical Analysis, Oxford University Press, 1993.
 
+    Examples
+    --------
+    >>> import numpy as np
+    >>> import matplotlib.pyplot as plt
+    >>> from scipy.interpolate import BSpline, spalde, splder, splev
+    
+    >>> # Store characteristic parameters of a Bspline =================================
+    >>> # (knots, coefficients, degree of the spline)
+    >>> tck = ((-2, -2, -2, -2, -1, 0, 1, 2, 2, 2, 2),
+    ...        (0, 0, 0, 6, 0, 0, 0),
+    ...        3)
+    
+    >>> # Instance a B-spline object ===================================================
+    >>> # `BSpline` objects are prefered, except for spalde()
+    >>> bspl = BSpline(tck[0], tck[1], tck[2])
+    >>> # Generate extra points to get a smooth curve ----------------------------------
+    >>> x = np.linspace(min(tck[0]), max(tck[0]), 100)
+    >>> x_ref = 1.25
+    
```

Comment:
Could you add some written text explaining why we are showing these three methods